pkg/compose/build.go

```diff
@@ -189,11 +189,14 @@ func (s *composeService) getLocalImagesDigests(ctx context.Context, project *typ
 		images[name] = info.ID
 	}
 
-	for _, s := range project.Services {
+	for i, s := range project.Services {
 		imgName := getImageName(s, project.Name)
```

Comment:
```suggestion
	for i := range project.Services {
		imgName := getImageName(project.Services[i], project.Name)
```

Minor suggestion - would prefer to avoid mixing and matching references within a given loop